.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -1,50 +1,48 @@
-name: "\U0001F41B Bug Report"
-description: File a bug report
+---
+name: "🐛 Bug Report"
+description: Report a bug
 title: "(short issue description)"
 labels: [bug, needs-triage]
+assignees: []
 body:
   - type: textarea
     id: description
     attributes:
       label: Describe the bug
-      description: A clear and concise description of what the bug is.
+      description: What is the problem? A clear and concise description of the bug.
     validations:
       required: true
-
   - type: textarea
     id: expected
     attributes:
-      label: Expected behavior
+      label: Expected Behavior
       description: |
-        Tell us what should happen.
+        What did you expect to happen?
     validations:
       required: true
-
   - type: textarea
     id: current
     attributes:
-      label: Current behavior
+      label: Current Behavior
       description: |
-        Tell us what happens instead of the expected behavior.
-
-        Include full errors, uncaught exceptions, stack traces, and relevant logs.
-        To turn on SDK logging, follow instructions here: http://docs.aws.amazon.com/sdk-for-java/v2/developer-guide/java-dg-logging.html
```

Comment:
Can we put back the link to the SDK logging instructions?

Reply:
New commit pushed